azure-toolkit-libs/azure-toolkit-sqlserver-lib/src/main/java/com/microsoft/azure/toolkit/lib/sqlserver/service/impl/SqlServer.java

```diff
@@ -113,56 +121,41 @@ public SqlServer commit() {
                 .withAdministratorLogin(getAdministratorLogin())
                 .withAdministratorPassword(getAdministratorLoginPassword())
                 .create();
-            // update inner property
-            SqlServer.this.sqlServerInner = server;
-            // update entity properties after created sql server successfully.
-            SqlServer.this.entity = SqlServer.this.fromSqlServer(server);
             // update
             if (isEnableAccessFromAzureServices() || isEnableAccessFromLocalMachine()) {
-                new SqlServerFirewallUpdater().withEnableAccessFromAzureServices(isEnableAccessFromAzureServices())
-                    .withEnableAccessFromLocalMachine(isEnableAccessFromLocalMachine()).commit();
+                SqlServer.this.update().commit();
             }
+            // refresh entity
+            SqlServer.this.refreshWebAppInner();
```

Comment:
why we need to refresh webapp inner?

Reply:
update Sql server entity and inner object once any property changed.